### ISSUE TITLE
Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type

### ISSUE THREAD

**Issue by kazunaritakeichi** [pulkitag/ief#7]
I tried `demo.py`.
I got the error below.

> I0708 12:26:11.132272 22880 net.cpp:413] Input 0 -> image
> I0708 12:26:11.132319 22880 net.cpp:413] Input 1 -> kp_pos
> I0708 12:26:11.132338 22880 net.cpp:413] Input 2 -> label
> I0708 12:26:11.132359 22880 layer_factory.hpp:77] Creating layer render1
> F0708 12:26:11.132725 22880 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Python (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, CropData, CroppingKat, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, EuclideanLossWithIgnore, Exp, Filter, Flatten, GenericWindowData, GenericWindowData2, GlobalEltwise, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, LRN, Log, MVN, MemoryData, MultinomialLogisticLoss, Normalize, PReLU, Pooling, Power, ROIPooling, RandomNoise, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, SmoothL1Loss, Softmax, SoftmaxWithLoss, Split, SquareBoxData, TanH, Threshold, Tile, Topography, WindowData)
> **\* Check failure stack trace: ***

I'm using the pulkitag/caffe.
I got no error with `make all`.
But I got the error below with `make test`.

> CXX src/caffe/test/test_data_layer.cpp
> /usr/bin/g++ src/caffe/test/test_data_layer.cpp -MMD -MP -pthread -fPIC -DCAFFE_VERSION=1.0.0-rc3 -DNDEBUG -O2 -DUSE_OPENCV -DUSE_LEVELDB -DUSE_LMDB -I/usr/include/python2.7 -I/usr/lib/python2.7/dist-packages/numpy/core/include -I/usr/local/include -I.build_release/src -I./src -I./include -I/usr/local/cuda-7.0/include -Wall -Wno-sign-compare -c -o .build_release/src/caffe/test/test_data_layer.o 2> .build_release/src/caffe/test/test_data_layer.o.warnings.txt \
>         || (cat .build_release/src/caffe/test/test_data_layer.o.warnings.txt; exit 1)
> src/caffe/test/test_roi_pooling_layer.cpp:19:38: fatal error: caffe/fast_rcnn_layers.hpp: No such file or directory
>  #include "caffe/fast_rcnn_layers.hpp"
>                                       ^
> compilation terminated.
> make: **\* [.build_release/src/caffe/test/test_roi_pooling_layer.o] ERROR 1

My environment is below.
ubuntu 14.04+CUDA7.0+cudnn4.0

Could you help me?
Thank you in advance.

**Comment by pulkitag** [comment]
Are you compiling with PYTHON_LAYER flag in your Makefile.config?

**Comment by kazunaritakeichi** [comment]
I compiled with PYTHON_LAYER flag.
I got no error.
Thank you.

**Comment by coocoky** [comment]
@ktak199 
how to compile with PYTHON_LAYER flag
